<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>settingsForm</class>
 <widget class="QWidget" name="settingsForm">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>500</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Ayarlar</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icons.qrc">
    <normaloff>:/resources/Weather-icon.png</normaloff>:/resources/Weather-icon.png</iconset>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <widget class="QPushButton" name="setupButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>470</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Kurulum</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Kontrol Toleransı</string>
   </property>
  </widget>
  <widget class="QSlider" name="errorLimitSlider">
   <property name="geometry">
    <rect>
     <x>152</x>
     <y>17</y>
     <width>100</width>
     <height>25</height>
    </rect>
   </property>
   <property name="minimum">
    <number>2</number>
   </property>
   <property name="maximum">
    <number>50</number>
   </property>
   <property name="pageStep">
    <number>5</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="labelErrorLimit">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>21</y>
     <width>30</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 255);</string>
   </property>
   <property name="text">
    <string>10</string>
   </property>
  </widget>
  <widget class="QPushButton" name="saveExitButton">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>420</y>
     <width>100</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 0, 0);</string>
   </property>
   <property name="text">
    <string>Kaydet &amp; Çık</string>
   </property>
  </widget>
  <widget class="QPushButton" name="exitButton">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>470</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>Çık</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="table">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>220</y>
     <width>381</width>
     <height>151</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOn</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="autoScroll">
    <bool>false</bool>
   </property>
   <property name="showGrid">
    <bool>true</bool>
   </property>
   <property name="gridStyle">
    <enum>Qt::SolidLine</enum>
   </property>
   <property name="cornerButtonEnabled">
    <bool>true</bool>
   </property>
   <property name="rowCount">
    <number>10</number>
   </property>
   <property name="columnCount">
    <number>1</number>
   </property>
   <attribute name="horizontalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>150</number>
   </attribute>
   <attribute name="horizontalHeaderHighlightSections">
    <bool>true</bool>
   </attribute>
   <attribute name="horizontalHeaderMinimumSectionSize">
    <number>150</number>
   </attribute>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderCascadingSectionResizes">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderDefaultSectionSize">
    <number>25</number>
   </attribute>
   <attribute name="verticalHeaderHighlightSections">
    <bool>true</bool>
   </attribute>
   <attribute name="verticalHeaderMinimumSectionSize">
    <number>25</number>
   </attribute>
   <row>
    <property name="text">
     <string>Kamera IP Adresi</string>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="background">
     <color>
      <red>0</red>
      <green>0</green>
      <blue>0</blue>
     </color>
    </property>
   </row>
   <row>
    <property name="text">
     <string>PLC IP Adresi</string>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
   </row>
   <row>
    <property name="text">
     <string>PLC Sağ Hareket Byte</string>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
   </row>
   <row>
    <property name="text">
     <string>PLC Sağ Hareket Bit</string>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
   </row>
   <row>
    <property name="text">
     <string>PLC Sol Hareket Byte</string>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
   </row>
   <row>
    <property name="text">
     <string>PLC Sol Hareket Bit</string>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
   </row>
   <row>
    <property name="text">
     <string>PLC Kumanda Stop Byte</string>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
   </row>
   <row>
    <property name="text">
     <string>PLC Kumanda Stop Bit</string>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
   </row>
   <row>
    <property name="text">
     <string>PLC Acil Stop Byte</string>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
   </row>
   <row>
    <property name="text">
     <string>PLC Acil Stop Bit</string>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
   </row>
   <column>
    <property name="text">
     <string>Value</string>
    </property>
    <property name="background">
     <color>
      <red>0</red>
      <green>0</green>
      <blue>0</blue>
     </color>
    </property>
   </column>
   <item row="0" column="0">
    <property name="text">
     <string/>
    </property>
   </item>
  </widget>
  <widget class="QSlider" name="yResIndexSlider">
   <property name="geometry">
    <rect>
     <x>152</x>
     <y>388</y>
     <width>100</width>
     <height>25</height>
    </rect>
   </property>
   <property name="minimum">
    <number>0</number>
   </property>
   <property name="maximum">
    <number>8</number>
   </property>
   <property name="singleStep">
    <number>1</number>
   </property>
   <property name="pageStep">
    <number>1</number>
   </property>
   <property name="sliderPosition">
    <number>0</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="labelyRes">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>391</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 255);</string>
   </property>
   <property name="text">
    <string>10</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>390</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Takip Trendi</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkPLConBoot">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>361</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Program Açıldığında PLC'ye Otomatik Bağlansın mı?</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkCamonBoot">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>381</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Program Açıldığında Kamera'ya Otomatik Bağlansın mı?</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>450</y>
     <width>400</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 0, 0);</string>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>2</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>130</y>
     <width>400</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 0, 0);</string>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>2</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QRadioButton" name="radio200">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>170</y>
     <width>82</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>S7-200</string>
   </property>
   <attribute name="buttonGroup">
    <string>radioGroup</string>
   </attribute>
  </widget>
  <widget class="QRadioButton" name="radio300">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>190</y>
     <width>82</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>S7-300</string>
   </property>
   <attribute name="buttonGroup">
    <string>radioGroup</string>
   </attribute>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>152</x>
     <y>190</y>
     <width>46</width>
     <height>13</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>DB No:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="editDBNo">
   <property name="geometry">
    <rect>
     <x>204</x>
     <y>187</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="inputMask">
    <string>999; </string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>setupButton</sender>
   <signal>clicked()</signal>
   <receiver>settingsForm</receiver>
   <slot>showSetupForm()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>89</x>
     <y>470</y>
    </hint>
    <hint type="destinationlabel">
     <x>378</x>
     <y>46</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>errorLimitSlider</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>settingsForm</receiver>
   <slot>errorLimitSlider()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>244</x>
     <y>23</y>
    </hint>
    <hint type="destinationlabel">
     <x>222</x>
     <y>26</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>saveExitButton</sender>
   <signal>clicked()</signal>
   <receiver>settingsForm</receiver>
   <slot>saveExitButton()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>239</x>
     <y>420</y>
    </hint>
    <hint type="destinationlabel">
     <x>295</x>
     <y>248</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>exitButton</sender>
   <signal>clicked()</signal>
   <receiver>settingsForm</receiver>
   <slot>exitButton()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>377</x>
     <y>475</y>
    </hint>
    <hint type="destinationlabel">
     <x>288</x>
     <y>275</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>yResIndexSlider</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>settingsForm</receiver>
   <slot>yResIndexSlider()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>223</x>
     <y>400</y>
    </hint>
    <hint type="destinationlabel">
     <x>213</x>
     <y>300</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkPLConBoot</sender>
   <signal>stateChanged(int)</signal>
   <receiver>settingsForm</receiver>
   <slot>checkPLConBoot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>303</x>
     <y>78</y>
    </hint>
    <hint type="destinationlabel">
     <x>339</x>
     <y>77</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkCamonBoot</sender>
   <signal>clicked()</signal>
   <receiver>settingsForm</receiver>
   <slot>checkCamonBoot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>331</x>
     <y>55</y>
    </hint>
    <hint type="destinationlabel">
     <x>369</x>
     <y>17</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radio200</sender>
   <signal>clicked()</signal>
   <receiver>settingsForm</receiver>
   <slot>changePLCtype()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>103</x>
     <y>178</y>
    </hint>
    <hint type="destinationlabel">
     <x>124</x>
     <y>178</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radio300</sender>
   <signal>clicked()</signal>
   <receiver>settingsForm</receiver>
   <slot>changePLCtype()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>93</x>
     <y>203</y>
    </hint>
    <hint type="destinationlabel">
     <x>145</x>
     <y>197</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>showSetupForm()</slot>
  <slot>errorLimitSlider()</slot>
  <slot>saveExitButton()</slot>
  <slot>exitButton()</slot>
  <slot>yResIndexSlider()</slot>
  <slot>checkPLConBoot()</slot>
  <slot>checkCamonBoot()</slot>
  <slot>changePLCtype()</slot>
 </slots>
 <buttongroups>
  <buttongroup name="radioGroup"/>
 </buttongroups>
</ui>
